packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/CheckboxesWidget.tsx

```diff
@@ -43,7 +42,7 @@ const CheckboxesWidget = ({ schema, label, id, disabled, options, value, require
 			{(get(uiSchema, ['ui:title']) || schema.title || label) && (
 				<KvFormLabel id={`${id}-title`} label={get(uiSchema, ['ui:title']) || schema.title || label} required={required} />
 			)}
-			<KvRadioButtonGroup buttons={buttons} selectedButtons={selectedButtons} onCheckedChange={onCheckedChange} disabled={disabled} />
+			<KvToggleButtonGroup buttons={buttons} selectedButtons={selectedButtons} onCheckedChange={onCheckedChange} />
```

Comment:
Since we removed the previous state of disabled from here, we should add support to it here. Add support to `disabledButtons` .